### ISSUE TITLE
Bug: CircleCI checkout fails when triggered on pull_request_review

### ISSUE THREAD

**Issue by kelvintaywl** [CircleCI-Public/trigger-circleci-pipeline-action#36]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I am not 100% sure this is a bug or perhaps more of a feature-request, but am filing an issue on behalf of a customer 🙇 

Given the following Github workflow config:

```yaml
name: Trigger CircleCI

on:
  pull_request_review:
    types: [submitted]
jobs:
  trigger-circleci:
    if: github.event.review.state == 'approved'
    runs-on: ubuntu-latest
    steps:
      - name: on PR review
        id: circleci_on_github_pr_review
        uses: CircleCI-Public/trigger-circleci-pipeline-action@v1.0.5
        env:
          CCI_TOKEN: ${{ secrets.CCI_TOKEN }}
```

When a PR is approved, this GitHub Action is triggered.
However, we see the following error on the subsequent CircleCI pipeline when `checkout` as the `branch` value for TriggerPipeline API is `refs/pull<PR number>/merge`:

```
Checking out branch
fatal: reference is not a tree: 74f97585e2e344a232fa48ea118bedf8ee6fd94f

exit status 128
```

Ref: https://app.circleci.com/pipelines/github/kelvintaywl-cci/circleci-gha-explore/8/workflows/cddf2112-2c4a-4828-986a-97dca5cb6e15/jobs/3?invite=true#step-101-81

Based on [the TriggerPipeline API documentation](https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline:~:text=or%20pull/%3Cnumber%3E/merge%20for%20the%20merge%20ref%20(GitHub%20only).), I wonder if the `branch` value should be:

```diff
- refs/pull/<PR number>/merge
+ pull/<PR number>/merge
```

### Minimum reproduction code

https://github.com/kelvintaywl-cci/circleci-gha-explore/pull/2

### Expected behavior

I am expecting that the CircleCI pipeline's job to be able to `checkout` successfully.

### GitHub Action Version

1.0.5

**Comment by KyleTryon** [comment]
Looks like any `pull_request_*` type event would be affected currently. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

**Comment by kelvintaywl** [comment]
thanks so much for investigating and making a fix so quickly #37 🙇 @KyleTryon 👏
